pkgs/servers/ldap/lldap/default.nix

```diff
@@ -98,6 +101,10 @@ in rustPlatform.buildRustPackage (commonDerivationAttrs // {
     substituteInPlace server/src/infra/tcp_server.rs --subst-var-by frontend '${frontend}'
   '';
 
+  postInstall = ''
+    mv $out/bin/migration-tool $out/bin/lldap_migration_tool
+  '';
```

Comment:
apparently, lldap had the same idea. so we'll have to remove this manual rename on next update

Reply:
Basically reverting this commit before the next lldap update? Or will the cargoBuildFlags stay?

Reply:
I think the cargo build flags will need to stay, otherwise only `lldap` is built